test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQConfigurationTests.cs

```diff
@@ -1,33 +1,32 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Trigger;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 using Moq;
 using Newtonsoft.Json;
 using RabbitMQ.Client;
-using Xunit;
-using System;
-using System.Collections.Generic;
-using Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Trigger;
 using RabbitMQ.Client.Events;
+using Xunit;
 
 namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ.Tests
 {
     public class RabbitMQConfigurationTests
     {
-        private static readonly IConfiguration _emptyConfig = new ConfigurationBuilder().Build();
-
         [Fact]
         public void Creates_Context_Correctly()
         {
             var options = new RabbitMQOptions { Hostname = "localhost", QueueName = "hello" };
             var loggerFactory = new LoggerFactory();
             var mockServiceFactory = new Mock<IRabbitMQServiceFactory>();
-            var config = new RabbitMQExtensionConfigProvider(new OptionsWrapper<RabbitMQOptions>(options), _emptyConfig, mockServiceFactory.Object, (ILoggerFactory)loggerFactory);
+            var mockNameResolver = new Mock<INameResolver>();

```

Comment:
Mocked the name resolver for now since other extensions use the common test framework